scripts/Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime
 # Create cron tasks & logfile
 RUN touch /var/log/cron.log
 COPY libsimple_crontab /etc/cron.d/circulation
+VOLUME /var/log
```

Comment:
This mounts the `/var/log`directory from inside the container to a container-specific location in the parent server. This will be *actually* helpful because this directory will have all of the logs from cron scripts.